app/scripts/factories/VariantFactory.js

```diff
@@ -584,12 +584,9 @@ angular.module('oncokbApp')
                 value: new Date().getTime()
             };
         }
-        function TimeStamp(userName, userEmail) {
-            this.by = {
-                name: userName,
-                email: userEmail
-            };
-            this.value = (new Date()).getTime().toString();
+        function Timestamp(username) {
```

Comment:
Why are we changing the signature? Also the structure of the timestamp? I did a quick search, `Timestamp` hasn't been used but the structure of the timestamp has been used, for instance, the VUS.

Reply:
I changed the structure since I feel like email is not necessary to be stored in firebase. We have emails of all uses in "User" collection so I think this is a kind of duplicate data. I will create another timestamp model, maybe call timestampVUS, to differentiate those 2 models.